### DISCUSSION TITLE
`<webview>` or `<browser>` support

### DISCUSSION THREAD

**Discussion by 0xENDER** [servo/servo#32920] [Ideas]
Hey, I'm just wondering if Servo supports the `<webview>` or the `<browser>` element. Does it? And if not, are there any plans to support any of the two?

**Comment by jdm** [comment]
We do not support those elements. There would need to be a strong motivating use case to consider it, I expect.

**Comment by 0xENDER** [comment]
Supporting these elements would ensure that developers can use this engine to the fullest. It'd enable developers to build full-on browsers using the engine without the need for any other tools/frameworks. And it could also enable developers to make custom frames with the content loaded in the `<webview>`/`<browser>` so it would be cleaner and enable a move controllable `<iframe>` variant.

**Comment by webbeef** [comment]
That's similar to https://github.com/servo/servo/discussions/32883 - Unite! :)
